admin_guide/configuring_authentication.adoc

```diff
@@ -421,20 +451,23 @@ oauthConfig:
     provider:
       apiVersion: v1
       kind: RequestHeaderIdentityProvider
-      clientCA: /path/to/client-ca.file <4>
-      headers: <5>
+      challengeURL: "https://www.example.com/challenging-proxy/oauth/authorize?${query}" <4>
+      loginURL: "https://www.example.com/login-proxy/oauth/authorize?${query}" <5>
+      clientCA: /path/to/client-ca.file <6>
```

Comment:
Since we're showing `challengeURL` and `loginURL` being set here, would it make more sense to have the example show `challenge` and `login` above it set to true?